web/src/components/CityContentFooter.tsx

```diff
@@ -1,19 +1,43 @@
 import React, { ReactElement } from 'react'
 import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
 
 import { DISCLAIMER_ROUTE, LICENSES_ROUTE, pathnameFromRouteInformation } from 'api-client'
 
 import buildConfig from '../constants/buildConfig'
 import CleanLink from './CleanLink'
 import Footer from './Footer'
 
-type PropsType = {
+const SidebarFooterContainer = styled.div`
+  width: 100%;
+  margin-top: -10px; /* to counteract the padding-top of the normal footer */
+  padding: 0 27px;
+  > * {
+    color: ${props => props.theme.colors.textColor};
+    padding: 16px 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-bottom: 1px solid #b1b1b1;
+    &:last-child {
+      border-bottom: none;
+    }
+  }
+`
+
+type CityContentFooterProps = {
   city: string
   language: string
   overlay?: boolean
+  inSidebar?: boolean
 }
 
-const LocationFooter: React.FC<PropsType> = ({ city, language, overlay = false }: PropsType): ReactElement => {
+const CityContentFooter: React.FC<CityContentFooterProps> = ({
+  city,
+  language,
+  overlay = false,
+  inSidebar = false,
```

Comment:
Optional:
What happens if overlay and inSidebar are set to true at the same time? `overlay` is just ignored. Therefore perhaps one property `mode: 'normal' | 'overlay' | 'sidebar' = 'normal'` would perhaps be better here.